### DISCUSSION TITLE
perfect_hands.ckpt

### DISCUSSION THREAD

**Discussion by Gloynus** [ssitu/ComfyUI_UltimateSDUpscale#33] [General]
You have the "perfect_hands.ckpt" model selected in your screenshot - where did you get it? Does its purpose correspond to its name?
![image](https://github.com/ssitu/ComfyUI_UltimateSDUpscale/assets/121782978/ff8019e8-8b59-47eb-ab3b-6bb45de15df0)

**Comment by ssitu** [comment]
it was a joke, it's just openjourney.
